interfaces/builtin/desktop_test.go

```diff
@@ -140,16 +159,11 @@ func (s *DesktopInterfaceSuite) TestMountSpec(c *C) {
 	restore := release.MockOnClassic(false)
 	defer restore()
 
-	// On all-snaps systems, the font related mount entries are missing
+	// On all-snaps systems, the mounts are present
 	spec := &mount.Specification{}
-	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.coreSlot), IsNil)
-	c.Check(spec.MountEntries(), HasLen, 0)
-
-	entries := spec.UserMountEntries()
-	c.Check(entries, HasLen, 1)
-	c.Check(entries[0].Name, Equals, "$XDG_RUNTIME_DIR/doc/by-app/snap.consumer")
-	c.Check(entries[0].Dir, Equals, "$XDG_RUNTIME_DIR/doc")
-	c.Check(entries[0].Options, DeepEquals, []string{"bind", "rw", "x-snapd.ignore-missing"})
+	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.appSlot), IsNil)
```

Comment:
do we need to keep some minimal testing for the coreSlot case here too?

Reply:
These checks have been moved down to lines 195-197. As I removed the early exit for `!release.OnClassic`, it seemed easier to put all the coreSlot tests together.